lang/test/org/partiql/lang/eval/EvaluatingCompilerGroupByTest.kt

```diff
@@ -932,4 +953,149 @@ EvaluatingCompilerGroupByTest : EvaluatorTestBase() {
             null)
     }
 
+    @Test
+    fun missingGroupByTest() {
+        checkInputThrowingEvaluationException(
+            "SELECT MAX(@v2), @v2 FROM `[1, 2.0, 3e0, 4, 5d0]` AS v2",
+            ErrorCode.EVALUATOR_VARIABLE_NOT_INCLUDED_IN_GROUP_BY,
+            mapOf(
+                    Property.LINE_NUMBER to 1L,
+                    Property.COLUMN_NUMBER to 19L,
+                    Property.BINDING_NAME to "v2"
+            )
+        )
+    }
+
+    @Test
+    fun missingGroupBySFWTest() {
+        checkInputThrowingEvaluationException(
+            """
+            SELECT x.title, MIN(money)
+            FROM books AS x, x.price AS money
+            WHERE author = 'Milan Kundera'
```

Comment:
This `WHERE` clause isn't important to this test, is it?

Reply:
The test was for SFW queries. I can remove the `WHERE` clause and/or the test if it seems too redundant.